apr2gus.txt

```diff
@@ -0,0 +1,2 @@
+PASS
+ok  	github.com/prebid/prebid-server	0.020s
```

Comment:
Can you remove this and the .swp file, please?

Reply:
removed